### PR TITLE
SHOWCASE-1146

### PR DESCRIPTION
**Pull request by parc-jason**
[Ensure the public call to read submission does not return status or notes.](https://apps.nrs.gov.bc.ca/int/jira/browse/SHOWCASE-1146)
Update email notification wording.

# Description

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I have checked that unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: app/src/db/migrations/20200521134300_007-settings-receipt-to-submission.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  return Promise.resolve()
+    .then(() => {
+      const config = {
+        template: 'confirmation-number-email.html',
+        from: 'MCAD.RegionalOps@gov.bc.ca',
+        subject: 'Industrial Camps Form Submission',
+        title: 'Industrial Camps Form Submission',
+        priority: 'normal',
+        messageLinkText: 'Click to view your Submission',
+        messageLinkUrl: 'https://comfort-dev.pathfinder.gov.bc.ca/app/minesoperatorscreening/review'
```
Comment (loneil) [inline app/src/db/migrations/20200521134300_007-settings-receipt-to-submission.js:13]: Reminder we'll need to update the DB in test and prod after the migrations to set these appropriately right?

File: app/src/forms/minesoperatorscreening/routes.js [hunk 2]
```diff
@@ -40,7 +40,7 @@ routes.put('/submissions/:submissionId', middleware.checkRole(['editor']), async
 });
 
 routes.get('/submissions/:submissionId', async (req, res, next) => {
-  await controller.readSubmission(req, res, next);
+  await controller.readSubmissionPublic(req, res, next);
```
Comment (loneil) [inline app/src/forms/minesoperatorscreening/routes.js:43]: We don't have anywhere in the app that needs to get everything from /submissions/1234 do we? So all calls to this just returing the "public" view are ok?
I think so, I think the inspection panel and notes panel fetch on their own from the sub routes.

File: app/src/db/migrations/20200521134300_007-settings-receipt-to-submission.js (hunk 1 above)
Comment (parc-jason) [reply app/src/db/migrations/20200521134300_007-settings-receipt-to-submission.js:13]: Correct.

File: app/src/forms/minesoperatorscreening/routes.js (hunk 2 above)
Comment (parc-jason) [reply app/src/forms/minesoperatorscreening/routes.js:43]: Yeah, what I checked in the FE does not use statusCodes/notes/etc. 
See transformToState in minesOperatorScreeningForm.
`  const primary = copy.contacts ? copy.contacts.find(({ contactType }) => contactType === 'PRIMARY') : {};
  const covid = copy.contacts ? copy.contacts.find(({ contactType }) => contactType === 'COVID_COORDINATOR') : {};
  copy.location.startDate = moment(copy.location.startDate).format('YYYY-MM-DD');
  copy.location.endDate = moment(copy.location.endDate).format('YYYY-MM-DD');
  return {
    business: copy.business,
    primaryContact: primary,
    covidContact: covid,
    attestation: copy.attestation,
    location: copy.location
  };
`

**Pull request merged**